src/main/java/net/dries007/tfc/common/blocks/devices/SheetPileBlock.java

```diff
@@ -60,15 +61,31 @@ public class SheetPileBlock extends ExtendedBlock implements EntityBlockExtensio
         .put(DOWN, box(0, 0, 0, 16, 1, 16))
         .build();
 
-    public static void removeSheet(Level level, BlockPos pos, BlockState state, Direction face, @Nullable Player player, boolean doDrops)
-    {
+
+    public static int faceToIndex(BlockState state, Direction face) {
```

Comment:
Please add a javadoc explaining the purpose of this method and why it's necessary, as it's not intuitive why we need a rotation + mirror saved here at all (except once you consider the effects of `rotate()` and `mirror()`.